web/src/main/resources/templates/educator/expressions-of-interest.html

```diff
@@ -1,11 +1,10 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
     <head th:replace="fragments/head"></head>
-    <link th:href="@{~/css/portal-styles.css}" rel="stylesheet"/>
-    <body onload="hidePositionsPendingTableIfEmpty()">
-    <!-- Top Navigation Bar-->
-    <div th:replace="fragments/navigation :: top-navigation"></div>
-    <body class="bg-dark bg-opacity-10">
+
+    <body class="bg-dark bg-opacity-10" onload="hidePositionsPendingTableIfEmpty()">
```

Comment:
Browser console error due to unused method call: `Uncaught ReferenceError: hidePositionsPendingTableIfEmpty is not defined
at onload (expressions-of-interest:33:85)`
```suggestion
    <body class="bg-dark bg-opacity-10">
```